local_deployment.py

```
@@ -2,46 +2,69 @@
-        if build == "stage" or build == "candidate" or build == "ga":
-            pull_stage_ga_images(version, build)
-            full_zip_name = pull_stage_ga_dependency_file(version, build)
+        if version_tuple < (8, 1, 0):
+            logging.info(f"Deploying MTA Version: {version} {build}")
+            if build == "stage" or build == "candidate" or build == "ga":
+                pull_stage_ga_images(version, build)
+                full_zip_name = pull_stage_ga_dependency_file(version, build)
+            else:
+                if not image_output_file:
+                    logging.info(f"Generating images list for {version}-{build}")
+                    image_list, stdout_err = generate_images_list(version, build)
```

Ruff (0.15.1)

[warning] Unpacked variable `stdout_err` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)